assets/sass/components/setup/_googlesitekit-setup.scss

```diff
@@ -107,6 +107,17 @@
 	.googlesitekit-setup__footer {
 		border-top: 1px solid $c-border-light;
 	}
+
+	.googlesitekit-setup__splash {
+		border-bottom: 1px solid #e6e6e6;
+		padding-left: 10px;
+		padding-right: 10px;
+	}
+
+	.googlesitekit-setup__splash .googlesitekit-setup__title {
```

Comment:
This should be simplified per @felixarntz in https://github.com/google/site-kit-wp/issues/2238#issuecomment-723280630

Reply:
Updated.